pytorch_lightning/trainer/connectors/optimizer_connector.py

```diff
@@ -25,7 +25,7 @@ def on_trainer_init(self, enable_pl_optimizer):
         self.trainer.optimizers = []
         self.trainer.optimizer_frequencies = []
 
-    def update_learning_rates(self, interval: str, monitor_metrics=None):
+    def update_learning_rates(self, interval: str, monitor_metrics=None, opt_indices=None):
```

Comment:
I think it will work for `step` level LR schedulers but we will need to call `update_learning_rates` separately as well for `epoch` level LR schedulers.

Reply:
I guess you could have an `on_epoch_end` hook in the `LightningOptimizer` for `epoch` level LR schedulers.